app/events/page.style.ts

```diff
@@ -0,0 +1,27 @@
+'use client';
+
+import NextImage from 'next/image';
+import styled from 'styled-components';
+import COLORS from '../../styles/colors';
+import { BespokeSans } from '../../styles/fonts';
+
+export const Image = styled(NextImage)`
+  layout: responsive;
+  width: 20%;
+  height: 90%;
+`;
+
+export const Page = styled.main`
+  background: ${COLORS.gray1};
+`;
+
+export const Title = styled.main`
```

Comment:
please use the heading available in `styles/text.ts`! to translate `rem` to `px`, usually you just have to multiply it by 16. in this case, since the desired font-size is `32px`, we want heading with font-size `2rem`, which corresponds to `H3`

or, for your component to inherit styles from the `<H3>` tag, you can do the following:

```typescript
import { H3 } from '../../styles/global'; // Import global components

// Extend H3 with additional styles
export const Title = styled(H3)`
  font-style: normal;
  line-height: normal;
  height: 50px;
`;
``` 

For additional context, the `<H3>` tag already has the Bespoke font set as its font-family, and it can accept props for `$fontWeight`, `$color`, and `$align`, so you can further adjust these as needed when using the Title component.